insalan/payment/admin.py

```diff
@@ -51,6 +54,27 @@ def has_delete_permission(self, _request, _obj=None):
         """Remove the ability to edit a payment from the backoffice"""
         return False
 
+    def export(self, request, queryset):
+        """
+        Export the selected payments to a CSV file
+        """
+        export = "id;prix;type;statut;date de paiement;date de dernière modification\n"
+        for payment in queryset:
+            # /!\ This will be removed when the payment timeouts are implemented
+            payment_status = payment.transaction.payment_status
+            if payment.transaction.payment_status == TransactionStatus.PENDING:
+                payment_status = TransactionStatus.FAILED
+            payment_type = ""
+            for product in payment.transaction.products.all():
+                payment_type += f"{product.name}, "
+            payment_type = payment_type[:-2]
```

Comment:
You probably could use something like `", ".join(payment.transaction.products.all())`